scripts/dashboard/Activity.vue

```diff
@@ -86,6 +87,23 @@ async function update(txToAdd = 0) {
     const orderedTxs = Array.from(wallet.getTransactions()).sort(
         (a, b) => a.blockHeight - b.blockHeight
     );
+
+    // For Rewards: aggregate the total amount
+    if (props.rewards) {
+        for (const tx of orderedTxs) {
+            // Only compute rewards
+            if (!tx.isCoinStake()) continue;
+            // If this Tx Height is over the last scanned height, we aggregate it
+            if (tx.blockHeight > nRewardUpdateHeight) {
+                // Aggregate the total rewards
+                rewardAmount.value += wallet.toHistoricalTXs([tx])[0].amount;
+            }
+        }
+        // Keep track of the scan block height
+        nRewardUpdateHeight = blockCount;
```

Comment:
I am not sure we should use `blockCount`... 
`orderedTxs[0].blockHeight` is safer (but make sure that the array has at least 1 element)

Reply:
Applied